### PR TITLE
[IOS-350]: Replace actionsheet with tableview for chat menus.

### PR DESCRIPTION
**Pull request by roberthein**
img_4294

### CODE REVIEW

File: Toshi/Controllers/Chat/ChatMenuTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import Foundation
+import UIKit
+import TinyConstraints
+
+class ChatMenuTableViewCell: UITableViewCell {
```
Comment (yuliaveres) [inline Toshi/Controllers/Chat/ChatMenuTableViewCell.swift]: please `final`

File: Toshi/Controllers/Chat/ChatMenuTableViewController.swift [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        tableView.separatorStyle = .none
+        tableView.register(ChatMenuTableViewCell.self, forCellReuseIdentifier: ChatMenuTableViewCell.reuseIdentifier)
+        tableView.reloadData()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        if let superview = view.superview?.superview {
+            superview.alpha = 0
```
Comment (yuliaveres) [inline Toshi/Controllers/Chat/ChatMenuTableViewController.swift]: why we need it ?

File: Toshi/Controllers/Chat/ChatMenuTableViewCell.swift (hunk 1 above)
Comment (roberthein) [reply Toshi/Controllers/Chat/ChatMenuTableViewCell.swift]: ✅

File: Toshi/Controllers/Chat/ChatMenuTableViewController.swift (hunk 2 above)
Comment (roberthein) [reply Toshi/Controllers/Chat/ChatMenuTableViewController.swift]: For a nice and smooth in-animation of the pop-over.

**Pull request merged**